style.css

```diff
@@ -0,0 +1,71 @@
+.box__link {
+	cursor: pointer;
+	color: black;
+	padding: 15px 10px;
+	text-decoration: none;
+	text-align: center;
+	font-size: 36px;
+}
+
+.menu__mobile {
+	position: relative;
+	display: none;
+	left: 0;
+	top: 5px;
+	content: '';
+	width: 25px;
+	height: 25px;
+	border-radius: 50%;
+	background: black;
+    opacity: 50%;
+ }
+
+.menu__mobile:before,
+.menu__mobile:after {
+	content: '';
+	display: block;
+	width: 20px;
+	height: 2px;
+	background: #FFF;
+	position: absolute;
+	left: 2px;
+} 
+
+.content {font-size: 36px;}
+ 
+@media screen and (max-width: 480px) {
+	
+	.box__link {display: none;}
+	
+	.menu__mobile {display: block;}
+  		
+	.menu__mobile::before {
+		position: relative;
+		left: 5px;
+		top: 9.25px;
+		content: '';
+		width: 15px;
+		height: 1px;
+		background: white;
+	}
+	.menu__mobile::after {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Azez1/pullRequest?prid=5948917)